be/src/olap/storage_engine.cpp

```diff
@@ -681,6 +686,10 @@ OLAPStatus StorageEngine::_start_trash_sweep(double* usage) {
     return res;
 }
 
+OLAPStatus StorageEngine::start_trash_sweep() {
```

Comment:
why add this function

Reply:
add a public function for users actively clean.

Reply:
since that function is not used by this pr, please remove this